README.md

```diff
@@ -1582,6 +1583,34 @@ In your other local project directory:
 yarn link @okta/okta-signin-widget
 ```
 
+### Utilizing Pseudo-loc
+
+> :warning: This tool requires access to Okta's internal registry via the VPN.
+
+A pseudo-localized language is a test language created to identify issues with the internationalization process. Generated from `login.properties` English resources, the pseudo-loc properties file can be used to test UI's for English leaks and CSS layout issues caused due to localization.
+
+To generate pseudo-loc, run the following command:
+
+```sh
+# Navigate into the pseudo-loc package
+[okta-siginin-widget]$ cd packages/@okta/pseudo-loc/
+
+# Install all required dependencies and generate login_ok_PL.propertiies
+# NOTE: This requires VPN access
+[pseudo-loc]$ yarn install
+[pseudo-loc]$ yarn pseudo-loc
+```
+
+Finally, update the `.widgetrc.js` file to use the `ok_PL` language, and start the [widget playground](#build-and-test-commands).
+
+```js
+module.exports = {
+  baseUrl: 'https://{yourOktaDomain}',
+  language: 'ok-PL',
```

Comment:
so the language would always be `ok-PL` whenever we need to run `pseudo-loc` ?

Reply:
This is for local testing purposes, following our guidance [here](https://github.com/okta/okta-signin-widget/#language-and-text) for passing in the language code.

Over time, we're planning to use pseudo localization as our test language (instead of English) - as it has a number of [ advantages when used during UI development](https://oktawiki.atlassian.net/wiki/spaces/~636081267/pages/1364887267/Globalization+g11n+Internationalization+i18n+and+Localization+l10n+Draft#What-should-developers-consider-during-design%3F). Happy to talk this over more if you're interested!